docs/mobility/slam/cartographer/_index.md

```diff
@@ -91,13 +104,49 @@ Creating a new map uses an instance of the cartographer module running in the cl
    2. Click **Create service** in the lower-left corner of the page.
       Choose `Data Management` as the type and specify a name for your data management service, for example `Data-Management-Service`.
    3. Click **Create**.
-   4. On the panel that appears, you can manage the capturing and syncing functions. You can also specify the **directory**, the sync **interval**, and any **tags** to apply to captured data. See the [data management service](/data/) for more information.
+   4. On the panel that appears, you can manage the capturing and syncing functions.
+      You can also specify the **directory**, the sync **interval**, and any **tags** to apply to captured data. See the [data management service](/data/) for more information.
 
-   Enable data capture for your camera, and for your movement sensor if you would like to use a movement sensor to provide IMU input:
+   Enable data capture for your camera, and for your movement sensor if you would like to use IMU and/or odometry input:
```

Comment:
This line renders outside of the stepwise procedural, but seems to be intended to introduce step 5 as a separate, per movement-sensor situation, scenario. Similar with the sentence before step 7.

I think you are introducing sections of steps outside of the steps, but keeping a continual growth of steps numbers through the whole process. I find that confusing, but I can start to see the intent if I consider. But what do you think of something like:

>Add the data management service:
>1. Steps
>2. More Steps.
>5. Enable data capture for your camera, and for your movement sensor if you would like to use IMU and/or odometry input:
>    - Find the card
>    - Click the button
>      - For Lidar
>      - For movement sensor
>         (tabset)
>
>Then, set up the `cartographer` module on your machine:
>
>1. Navigate to the Config tab ...
>2. Click the Services subtab ...

Nit: I think we try to avoid `and/or`. I'm fine with it, but if we want an alternative, maybe:

```suggestion
   Enable data capture for your camera, and for your movement sensor if you would like to use IMU input, odometry input, or both:
```

Reply:
Yes, it is a bit odd but I was trying not to refactor the whole thing and rather to just add the needed info. Coming up with a new format--lmk what you think of the next commit!

Reply:
And yes, applying the "or both" bit instead of and/or